src/pytest_postgresql/plugin.py

```diff
@@ -33,6 +33,7 @@
 _help_unixsocketdir = "Location of the socket directory"
 _help_dbname = "Default database name"
 _help_load = "Load this SQL file by default, may be set multiple times"
+_help_postgres_start = 'Postgres executable extra parameters'
```

Comment:
let's mention and indicate, that this is for the -o parameter, options. Here and in readme's table.

Reply:
This should be addressed now